test/mastodon-tl-tests.el

```diff
@@ -266,7 +266,7 @@ a string or a numeric."
  | Account 42 (@acct42@example.space) 2999-99-99 00:11:22
   ------------"))
 	(should (eq (get-text-property handle-location 'mastodon-tab-stop byline)
-                'shr-url))
+                'handle))
```

Comment:
We should probably also assert that there is the expected property with the account data (handle name at the moment) at the point.